src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/SnapshotTracker.kt

```diff
@@ -253,6 +254,9 @@ class SnapshotTracker {
         )
     }
 
+    private fun shouldIgnoreClass(owner: String) =
+        owner.startsWith("com.intellij.rt.debugger")
```

Comment:
Could you check if this is still required?
I've added `com.intellij.rt.debugger` into a list of non-instrumented packages in some previous PR:
https://github.com/JetBrains/lincheck/blob/7b02d5410ca4952bdb07594f8c0e00377ae5b5d7/src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckJavaAgent.kt#L509

Reply:
I had this problem in the original PR, where the check in agent's `shouldTransform` was present already.
There was an attempt to add over 50k+ allocated objects in the snapshot, when I was debugging some code. Right now I am not able to reproduce it (when I remove `shouldIgnoreClass` method)

Reply:
checked for the second time -- it helped